diff -urNad gosa-core~/include/functions.inc gosa-core/include/functions.inc
--- gosa-core~/include/functions.inc    2010-02-02 14:02:18.000000000 +0100
+++ gosa-core/include/functions.inc     2010-02-02 14:01:25.000000000 +0100
@@ -40,7 +40,7 @@
     e.g.  SetEnv CONFIG_FILE gosa.conf.2.6
  */
 if(!isset($_SERVER['CONFIG_FILE'])){
-  define ("CONFIG_FILE", "gosa.conf");
+  define ("CONFIG_FILE", "/usr/share/doc/gosa");
 }else{
   define ("CONFIG_FILE",$_SERVER['CONFIG_FILE']);
 }

